Bottom: 35df34c0d1839221f8efd58936e3a13861bea590
Top:    bd1eba46dc9f37a0419795f379abe6123e56b8dc
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 14:49:27 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index ec22fa0..5a09bf6 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1348,11 +1348,6 @@ static void *statemachine(void *arg)
 		case SIGINT:
 		case SIGUSR2:
 			master_mutex_lock();
-			while (!list_empty(&master_list->completed)) {
-				master_mutex_unlock();
-				finish_cond_wait();
-				master_mutex_lock();
-			}
 			if (list_empty(&master_list->mounts)) {
 				master_mutex_unlock();
 				return NULL;
